examples/grpc/yclient/main.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"go.uber.org/yarpc"
+	"go.uber.org/yarpc/encoding/raw"
+	"go.uber.org/yarpc/transport"
+	"go.uber.org/yarpc/transport/grpc"
+
+	"golang.org/x/net/context"
+)
+
+func main() {
+	dispatcher := yarpc.NewDispatcher(yarpc.Config{
+		Name: "hello",
+		Outbounds: transport.Outbounds{
+			"yarpc": grpc.NewOutbound("localhost:50054"),
```

Comment:
Not sure, this is what they use in their own examples when they "grpc.Dial" into downstreams.

https://github.com/grpc/grpc-go/blob/master/examples/route_guide/client/client.go#L56